stl/parser.py

```diff
@@ -252,7 +253,13 @@ def p_message_def(p):
       msg = stl.message.Message(p[2], encode_name, p[1])
       msg.fields, msg.messages = p[5]
     else:  # EXTERNAL STRING_LITERAL ';'
-      msg = stl.message.MessageFromExternal(p[2], encode_name, p[1], p[5])
+      try:
+        msg = stl.message.MessageFromExternal(p[2], encode_name, p[1], p[5])
+      except Exception as e:
+        logging.error('Could not import message: %s', p[5])
```

Comment:
You can use logging.exception here instead and it will include the traceback automagically